templates/eightball/views/error.tsx

```
@@ -0,0 +1,33 @@
+import React from 'react'
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)